docs/source/build-docs.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+DOCS_DIR="./docs"
+VENV_DIR="$(mktemp -d)"
```

Comment:
frankly... I'd rather be able to get a fixed venv name in order to be able to re-use it. It'd be faster, and would avoid re-builting the venv each time when you're nit-picking on a local change.

Reply:
I agree in principal. It does take more time, however keeping the venv does have some drawbacks when you are making changes to the build process, as the old venv may interfere with your changes.

That being said it is a personal preference, and if you want a venv with a longer life time I have no problem.